re_data_ui/src/utils/helpers.ts

```diff
@@ -107,6 +107,12 @@ export const generateModelId = (details: DbtNode | DbtSource): string => {
   const {
     database, schema,
   } = details;
-  const identifier = details.resource_type === 'source' ? (details as DbtSource).identifier : details.name;
+  let identifier;
+  if (details.resource_type === 'source') {
+    const detail = details as DbtSource;
```

Comment:
I think naming it detail (just removing s) may be a bit confusing 😅

Reply:
a bit of nit picking

Reply:
I was trying to avoid [eslint/no-param-reassign](https://eslint.org/docs/rules/no-param-reassign) 😅

Reply:
that works, thanks!